source/source.json

```
@@ -5113,6 +5113,9 @@
 			"Angband-(.*)-3ds\\.zip": {
 				"Angband-{}.3dsx": ["Angband-{}.3dsx", "angband"]
 			},
+			"Angband-(.*)-3ds\\.zip": {
+				"Angband-{}.cia": ["Angband-{}.cia", "angband"]
+			},
```

Since they're in the same ZIP I think this can be simplified to this:```suggestion
			"Angband-(.*)-3ds\\.zip": {
				"Angband-{}.3dsx": ["Angband-{}.3dsx", "angband"]
				"Angband-{}.cia": ["Angband-{}.cia", "angband"]
			},
```
Might actually be a JSON duplicate key error as is, but I didn't check